fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -192,21 +194,45 @@ public Response options() {
      * Retrieve the node profile
      *
      * @param rangeValue the range value
-     * @return triples for the specified node
+     * @return the binary or the triples for the specified node
      * @throws IOException if IO exception occurred
      */
     @GET
-    @Produces({TURTLE + ";qs=10", JSON_LD + ";qs=8",
-            N3, N3_ALT2, RDF_XML, NTRIPLES, APPLICATION_XML, TEXT_PLAIN, TURTLE_X,
-            TEXT_HTML, APPLICATION_XHTML_XML, "*/*"})
+    @Produces({TURTLE + ";qs=10", "*/*"})
     public Response describe(@HeaderParam("Range") final String rangeValue) throws IOException {
         checkCacheControlHeaders(request, servletResponse, resource(), session);
 
         LOGGER.info("GET resource '{}'", externalPath);
+
+        final ImmutableList<MediaType> acceptableMediaTypes =
+                new ImmutableList.Builder<MediaType>().addAll(headers.getAcceptableMediaTypes()).build();
```

Comment:
No, I don't think so, but I was trying to prevent a few function calls to access that list. I could just call `headers.getAcceptableMediaTypes()` whenever I need the list, or just use a 
```java
final List<MediaType> acceptableMediaTypes = headers.getAcceptableMediaTypes();
```
Which way is preferred in Java?